doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -70,5 +70,4 @@ Science time!
 
 Now you can proceed with running ISR and other "processCcd" tasks via a Gen3 pipeline.
 
-Some useful pipelines can be found in the ``pipelines/DarkEnergyCamera`` directory of the :py:mod:`lsst.ap.pipe` package.
-These and other pipelines may move to ``recipes/DarkEnergyCamera`` as `RFC-775 <https://jira.lsstcorp.org/browse/RFC-775>`__ is implemented.
+Some useful pipelines can be found in the ``pipelines/DECam`` directory of the :py:mod:`lsst.ap.pipe` package.
```

Comment:
To avoid bias, we might also mention `lsst.drp.pipe` (which was planned but did not yet exist at the time of writing).